typed/typed.go

```diff
@@ -291,3 +291,13 @@ func (c *Comparison) String() string {
 	}
 	return bld.String()
 }
+
+// Remove fields from the compare
```

Comment:
Maybe make the name and description a little clearer. "Remove" is not clear enough, especially since it has a field name removed, I want to make sure we don't forget what this is.